daprdocs/content/en/developing-applications/building-blocks/actors/howto-actors.md

```diff
@@ -129,3 +129,16 @@ DELETE http://localhost:3500/v1.0/actors/<actorType>/<actorId>/reminders/<name>
 ```
 
 Refer [api spec]({{< ref "actors_api.md#invoke-reminder" >}}) for more details.
+
+## Actor Runtime Configuration
+
+You can configure the Dapr Actors runtime configuration to modify the default runtime behavior.
+
+### Current defaults
+- `actorIdleTimeout` The timeout before deactivating an idle actor. Checks for timeouts occur every `actorScanInterval` interval. **Default: 60 minutes**
+- `actorScanInterval` The duration which specifies how often to scan for actors to deactivate idle actors. Actors that have been idle longer than actor_idle_timeout will be deactivated. **Default: 30 seconds**
+- `drainOngoingCallTimeout` The duration when in the process of draining rebalanced actors. This specifies the timeout for the current active actor method to finish. If there is no current actor method call, this is ignored. **Default: 60 seconds**
+- `drainRebalancedActors` If true, Dapr will wait for `drainOngoingCallTimeout` duration to allow a current actor call to complete before trying to deactivate an actor. **Default: true**
+- `reentrancy (ActorReentrancyConfig)` Configure the reentrancy behavior for an actor. If not provided, reentrancy is diabled. **Default: disabled**
+
+Refer to the [Dapr SDKs]({{< ref "developing-applications/sdks/#sdk-languages" >}}) for more details.
```

Comment:
It'd be good to also refer to this page https://docs.dapr.io/reference/api/actors_api/#get-registered-actors.

Maybe provide an example of a full JSON config there?

Reply:
@halspang I am not sure what you are thinking wrt the get registered actors endpoint. Can you just write which ever sentence I should include?

Re JSON config: It's a bit weird to provide an example here because the SDKs don't create configs FROM JSON. I didn't want to favor one of the SDKs over another in this spot and introduce a place that could become outdated.

What do you think?

Reply:
This is the endpoint the provides the actual config. In the SDKs they're just setting the object values that's serialized into JSON in this endpoint. The thought here was just to raise awareness around what's actually being called behind the scenes.

If we're trying to not bias towards an SDK though, maybe not bias towards how to do this with a 'raw' actor either? The real concern was just that it's not really clear how these values are really set in dapr itself.

@msfussell, what are your thoughts on how we should talk about how actors function? Do we want to only talk about the SDK path or do we want customers making actors directly as a lot of our SDKs don't have actor support anyway?

Reply:
@msfussell ping on this

Reply:
Actors are primarily used by SDKs, but that does not prevent the use of the APIs where an SDK is not supported. Hence we are trying to be as informative as possible for all Actor settings.

Reply:
I'd personally argue that actors are unlike other building block APIs, in that they require SDKs to operate due to the complexity of managing them. I'd prefer not to document the APIs as a supported method for now.